src/integrationTest/java/uk/gov/hmcts/reform/lrdapi/RetrieveRegionDetailsIntegrationTest.java

```diff
@@ -52,19 +106,63 @@ public void doesNotReturnRegionDetailsByInvalidDescriptionWithStatusCode_404() t
     }
 
     @Test
-    public void doesNotReturnRegionDetailsByMissingDescriptionWithStatusCode_400() throws JsonProcessingException {
+    public void doesNotReturnRegionByInvalidDescriptionSpecialCharWithStatusCode_400() throws JsonProcessingException {
 
         Map<String, Object> errorResponseMap = (Map<String, Object>)
-            lrdApiClient.findRegionDetailsByDescription("", ErrorResponse.class);
+            lrdApiClient.findRegionDetailsByDescription("Lo£nd*on", ErrorResponse.class);
 
 
         assertThat(errorResponseMap).containsEntry(HTTP_STATUS, HttpStatus.BAD_REQUEST);
     }
 
-    private void responseVerification(LrdRegionResponse response) {
-        assertThat(response.getRegionId()).isEqualTo("2");
-        assertThat(response.getDescription()).isEqualTo("London");
-        assertThat(response.getWelshDescription()).isNull();
+    @Test
+    public void doesNotReturnRegionByInvalidIdnSpecialCharWithStatusCode_400() throws JsonProcessingException {
+
+        Map<String, Object> errorResponseMap = (Map<String, Object>)
+            lrdApiClient.findRegionDetailsByDescription("1*", ErrorResponse.class);
+
+
+        assertThat(errorResponseMap).containsEntry(HTTP_STATUS, HttpStatus.BAD_REQUEST);
+    }
+
+    private void responseVerification(List<LrdRegionResponse> response, int expectedRegions) {
+        assertThat(response.size()).isEqualTo(expectedRegions);
+        assertThat(response.get(0).getRegionId()).isEqualTo("2");
+        assertThat(response.get(0).getDescription()).isEqualTo("London");
+        assertThat(response.get(0).getWelshDescription()).isNull();
+        if (expectedRegions == 2) {
+            assertThat(response.get(1).getRegionId()).isEqualTo("3");
+            assertThat(response.get(1).getDescription()).isEqualTo("Midlands");
+            assertThat(response.get(1).getWelshDescription()).isNull();
+        }
+    }
+
+    private void responseVerificationForAll(List<LrdRegionResponse> response) {
+        assertThat(response.size()).isEqualTo(8);
+        assertThat(response.get(0).getRegionId()).isEqualTo("2");
```

Comment:
I think it looks more cleaner if the expected response is prepared in a list and the assertj library is used like below:
```assertThat(actual).hasSize(expected.size()).hasSameElementsAs(expected);```